### PR TITLE
New package: python-rapidfuzz

### PR DESCRIPTION
**Pull request by wlph17**

### CODE REVIEW

File: mingw-w64-python-rapidfuzz/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  "${MINGW_PACKAGE_PREFIX}-python"
+  "${MINGW_PACKAGE_PREFIX}-python-pandas"
+  "${MINGW_PACKAGE_PREFIX}-python-numpy"
+  "${MINGW_PACKAGE_PREFIX}-cc"
+)
+makedepends=("${MINGW_PACKAGE_PREFIX}-python-setuptools")
+checkdepends=("${MINGW_PACKAGE_PREFIX}-python-pytest-cov"
+              "${MINGW_PACKAGE_PREFIX}-python-pytest-runner")
+source=("https://github.com/maxbachmann/RapidFuzz/archive/refs/tags/v${pkgver}.tar.gz"
+        "https://github.com/maxbachmann/rapidfuzz-cpp/archive/103674d.tar.gz"
```
Comment (Biswa96) [inline mingw-w64-python-rapidfuzz/PKGBUILD:23]: These external tarball are not required if source tarball is downloaded from pypi. See the source link here https://github.com/archlinux/svntogit-community/blob/packages/python-rapidfuzz/trunk/PKGBUILD

Comment (wlph17) [reply mingw-w64-python-rapidfuzz/PKGBUILD:23]: But it has no tests to do pytest in the file downloaded from pypi, is that ok?

Comment (Biswa96) [reply mingw-w64-python-rapidfuzz/PKGBUILD:23]: Somewhat OK. Tests are not done in CI and it's python - _write once, run everywhere_ :)

Comment (Biswa96) [reply mingw-w64-python-rapidfuzz/PKGBUILD:23]: Just kidding. This package has C module, so no.

Comment (wlph17) [reply mingw-w64-python-rapidfuzz/PKGBUILD:23]: so, no change?

Comment (Biswa96) [reply mingw-w64-python-rapidfuzz/PKGBUILD:23]: That was just my suggestion. You can use source tarball from pypi to replace those external tarball related steps.

Comment (wlph17) [reply mingw-w64-python-rapidfuzz/PKGBUILD]: ok

**Pull request merged**